test/ShallowWrapper-spec.js

```diff
@@ -3056,7 +3056,7 @@ describe('shallow', () => {
         }
       }
       shallow(<Foo />, { lifecycleExperimental: false });
-      expect(spy.calledOnce).to.equal(false);
+      expect(spy.called).to.equal(false);
```

Comment:
`expect(spy).to.have.property('callCount', 0)` will have a nicer failure message